### DISCUSSION TITLE
[Feature Request] Translation of the extension

### DISCUSSION THREAD

**Discussion by guiga-zalu** [EnixCoda/Gitako#199] [Ideas]
I think it'd be beneficial for the Gitako extension to have translations for the interface.

Even though most texts in the interface are "Search files", "path", the pin tool tip and the settings menu, having these only in English might still manifest a language barrier.

If it's decided that translations should be made, I'm open to translate text strings to Portuguese, and I'm sure others would be happy to translate it to other languages.

**Comment by EnixCoda** [comment]
Thanks @guiga-zalu 

I'm curious whether GitHub shows its content in Portuguese when you set your browser language to i Portuguese?

I'll agree with providing translations of the languages that GitHub supports.

**Reply by guiga-zalu** [reply]
As of [18/11/2016](https://github.blog/2010-07-13-github-in-your-language/), GitHub supports only English.

**Comment by TC999** [comment]
There is a Github Chinese plug-in on Github. After installing the plug-in, when the browser language is Chinese, the plug-in will automatically Chineseize the Github components. Other languages are not clear.
